docs/dev/guidelines/client.rst

```diff
@@ -35,6 +35,12 @@ Some general aspects:
 2. Components
 =============
 
+In our project, we advocate for the creation of standalone components. A standalone component is a self-contained unit that encapsulates its own logic, view, and styles. It doesn't directly depend on its parent or child components and can be reused in different parts of the application.
```

Comment:
Could we explicitly mention modules as the alternative since it's currently what we have in Artemis?